homeassistant/components/satel_integra/__init__.py

```diff
@@ -68,13 +93,20 @@
 
     zones = conf.get(CONF_ZONES)
     outputs = conf.get(CONF_OUTPUTS)
+    switchable_outputs = conf.get(CONF_SWITCHABLE_OUTPUTS)
     host = conf.get(CONF_DEVICE_HOST)
     port = conf.get(CONF_DEVICE_PORT)
-    partition = conf.get(CONF_DEVICE_PARTITION)
+    partitions = conf.get(CONF_DEVICE_PARTITIONS)
 
     from satel_integra.satel_integra import AsyncSatel
 
-    controller = AsyncSatel(host, port, hass.loop, zones, outputs, partition)
+    monitored_outputs = collections.OrderedDict(
+        list(outputs.items()) +
```

Comment:
We should be able to use `dict.update`, right?

Reply:
As agreed on the channel, will stay with original construct.